source/index.html

```diff
@@ -1,30 +1,258 @@
+            <li><h4>Company</h4></li>
+            <li>
+              <a href="#about">About</a>
+            </li>
+            <li>
+              <a href="resume.html">Resume</a>
+            </li>
+            <li>
+              <a href="#projects">Projects</a>
+            </li>
+            <li>
+              <a href="#contact">Contact</a>
+            </li>
+          </ul>
+        </div>
+        <div class="footer-menu">
+          <ul>
+            <li><h4>Links</h4></li>
+            <li>
+              <a href="https://www.facebook.com/" target="_blank">Facebook</a>
+            </li>
+            <li>
+              <a href="https://www.instagram.com/" target="_blank">Instagram</a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/feed/" target="_blank">LinkedIn</a>
+            </li>
+          </ul>
+        </div>
+        <div class="footer-menu">
+          <ul>
+            <li><h4>Get in touch</h4></li>
+            <li><a href="mailto:mark.m.hayes1@gmail.com">Email@me</a></li>
+            <li><a href="tel:906-322-3097">Call Me</a></li>
+            <li>223 Genesee St.<br>Marqutette, MI 49855</li>
+          </ul>
+        </div>
+      </div>
+      <hr>
+      <p>Pot Pie Digital, LLC 223 Genesee St., Marquette, Mi 49855. 906-322-3097</p>
+    </footer>
+  </main>
+  <script src="./main.js " defer></script>
+  <script
+  src="https://code.jquery.com/jquery-3.3.1.min.js"
+  integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
+  crossorigin="anonymous"></script>
+  <script type="text/javascript">
+          // Select all links with hashes
+      $('a[href*="#"]')
+        // Remove links that don't actually link to anything
+        .not('[href="#"]')
+        .not('[href="#0"]')
+        .click(function(event) {
+          // On-page links
+          if (
+            location.pathname.replace(/^\//, '') == this.pathname.replace(/^\//, '')
+            &&
+            location.hostname == this.hostname
+          ) {
+            // Figure out element to scroll to
+            var target = $(this.hash);
+            target = target.length ? target : $('[name=' + this.hash.slice(1) + ']');
+            // Does a scroll target exist?
+            if (target.length) {
+              // Only prevent default if animation is actually gonna happen
+              event.preventDefault();
+              $('html, body').animate({
+                scrollTop: target.offset().top
+              }, 1000, function() {
+                // Callback after animation
+                // Must change focus!
+                var $target = $(target);
+                $target.focus();
+                if ($target.is(":focus")) { // Checking if the target was focused
+                  return false;
+                } else {
+                  $target.attr('tabindex','-1'); // Adding tabindex for elements not focusable
+                  $target.focus(); // Set focus again
+                };
```

Comment:
Everything from `// Callback after animation` to this line is what's causing the weird outline, and you're right about it being the default `:focus` styling. I don't think you need to focus on these links as you navigate to them though (it's not standard behavior to do that) so probably safe to just remove the contents of this callback.